src/AccessibilityInsights.Extensions.AzureDevOps/AzureDevOpsIntegration.cs

```diff
@@ -48,6 +49,8 @@ public partial class AzureDevOpsIntegration
         /// </summary>
         public bool ConnectedToAzureDevOps => _baseServerConnection?.HasAuthenticated == true;
 
+        public ExtensionConfiguration Configuration { get; private set; } = new ExtensionConfiguration();
```

Comment:
>private set; [](start = 59, length = 12)

The private setter is only required if this member gets changed after construction. From what I see in the code, that's not happening, so I think the private setter can go away.

Reply:
It is gone